config/config.json

```diff
@@ -103,16 +103,20 @@
         "bugprone-bool",
         "bugprone-copy",
         "bugprone-dangling",
+        "bugprone-dynamic-static-initializers",
         "bugprone-fold",
         "bugprone-forward",
         "bugprone-inaccurate",
+        "bugprone-infinite-loop",
```

Comment:
Please keep the keys also in alphabetical order in this file too. Check the other new keys too.